emulators/mednafen/Portfile

```diff
@@ -3,63 +3,51 @@
 PortSystem          1.0
 
```

Comment:
Please use the legacysupport portgroup here, i.e. add

PortGroup    legacysupport   1.0

This will fix the clock_gettime issue